pom.xml

```diff
@@ -487,7 +487,7 @@
         <version.org.eclipse.microprofile.rest.client.api>3.0</version.org.eclipse.microprofile.rest.client.api>
         <version.org.eclipse.yasson>3.0.1</version.org.eclipse.yasson>
         <legacy.version.org.eclipse.persistence.eclipselink>2.7.11</legacy.version.org.eclipse.persistence.eclipselink>
```

Comment:
Is this still used anywhere in the build?

Reply:
Just in https://github.com/wildfly/wildfly/blob/main/boms/legacy-ee/pom.xml#L68

Reply:
I'll remove it